backend/src/database/ExtendedSQLite.js

```diff
@@ -1,16 +1,29 @@
-import { SQLite } from '@hocuspocus/extension-sqlite'
-import sqlite3 from 'sqlite3'
+// ExtendedSQLite.js
+import { SQLite } from '@hocuspocus/extension-sqlite';
+import sqlite3 from 'sqlite3';
 
 export class ExtendedSQLite extends SQLite {
   constructor(configuration) {
-    super(configuration)
-    this.initializeDatabase()
+    super(configuration);
+    this.initializeDatabase();
   }
 
   async onConfigure() {}
 
   initializeDatabase() {
-    this.db = new sqlite3.Database(this.configuration.database)
-    this.db.run(this.configuration.schema)
+    this.db = new sqlite3.Database(this.configuration.database, (err) => {

```

Comment:
Great thing btw, thanks for added.

But it won't work, because you've skipped the second argument (MODE).